code/modules/mob/living/carbon/superior_animal/roach/roach.dm

```diff
@@ -16,14 +16,17 @@
 	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat/roachmeat
 	meat_amount = 2
 
-	maxHealth = 10
-	health = 10
+	maxHealth = 20
+	health = 20
```

Comment:
This may be a tad overbuffed, considering there should be more roaches around due to the revive and food changes.

Reply:
15-15 now, so you can 1 shot it with a crowbar still, good point out